exercises/atbash-cipher/example.ml

```diff
@@ -0,0 +1,57 @@
+let substitute = function
+  | 'a' .. 'z' as c ->
+    let offset = (Char.code c) - (Char.code 'a') in
+    let code = (Char.code 'z') - offset in
+    Char.chr code
+  | other -> other
+
+let is_encodable = function
+  | 'a' .. 'z' | '0' .. '9' -> true
+  | _ -> false
+
+let explode s =
```

Comment:
I have troubles providing an inline signature because of the use of `function`. Because it is an implementation detail, I can't provide the signature in the `mli`-file.

Do you have a tip?